rvgo/fast/vm.go

```diff
@@ -7,22 +7,65 @@ import (
 )
 
 const (
-	fdStdin         = 0
-	fdStdout        = 1
-	fdStderr        = 2
-	fdHintRead      = 3
-	fdHintWrite     = 4
-	fdPreimageRead  = 5
-	fdPreimageWrite = 6
+	SysExit             = 93
```

Comment:
How about moving this constants to separate file? At `rvgo` dir's root,

Reply:
Created a `riscv` package and moved common constants to it.